web/js/map-OpenLayers.js

```diff
@@ -844,6 +847,9 @@ OpenLayers.Format.FixMyStreet = OpenLayers.Class(OpenLayers.Format.JSON, {
         if (typeof(obj.pagination) != 'undefined') {
             $('.js-pagination').html(obj.pagination);
         }
+        // Remove loading indicator
```

Comment:
I wonder if this should be higher (just before the `if typeof json == 'string'` line) so that if something goes wrong with the HTML setting (don't think this has happened yet), the person isn't stuck with the spinner.